clang/test/Driver/ftarget-compile-fast.cpp

```diff
@@ -1,20 +1,21 @@
 // Test -ftarget-compile-fast behaviors
```

Comment:
Comment can be updated to mention the use of new offload driver

Reply:
The files where an 'old-model' variant was added I do not update the comments to reflect.  The naming should take care of that.  Once the old model goes away, all of these tests will also be updated to remove the `--offload-new-driver` options being set.